R/topics.R

```diff
@@ -92,9 +92,20 @@ match_env <- function(topics) {
         purrr::map_lgl(any)
 
       which(match & is_public(internal))
+    },
+    none_of_concepts = function(x, internal = FALSE) {
```

Comment:
I _think_ the most natural English inverse of `has_concept()` would be `lacks_concept()`.

Reply:
God proposal. Thanks.